packages/react-composites/src/composites/CallWithChatComposite/adapter/CallWithChatBackedChatAdapter.ts

```diff
@@ -41,7 +43,23 @@ export class CallWithChatBackedChatAdapter implements ChatAdapter {
   ): Promise<void> {
     await this.callWithChatAdapter.sendMessage(content, /* @conditional-compile-remove(file-sharing-acs) */ options);
   };
+  /* @conditional-compile-remove(rich-text-editor-image-upload) */
+  uploadImageHandler = async function (
+    this: CallWithChatBackedChatAdapter,
```

Comment:
Maybe a different name to `this`?

Reply:
> Maybe a different name to `this`?

Agreed. However, it seems to be the convention in this file. Every handler is using `this`